ocf_datapipes/production/xgnational.py

```diff
@@ -0,0 +1,45 @@
+"""
+Pipeline to return live GSP and NWP data
+"""
+
+import logging
+from pathlib import Path
+from typing import Union
+
+import xarray
+
+import ocf_datapipes  # noqa
+from ocf_datapipes.config.load import load_yaml_configuration
+from ocf_datapipes.config.model import Configuration
+from ocf_datapipes.load import OpenGSPFromDatabase, OpenNWP
+
+logger = logging.getLogger(__name__)
+xarray.set_options(keep_attrs=True)
+
+
+def xgnational_production_datapipe(configuration_filename: Union[Path, str]) -> dict:
```

Comment:
So this datapipe is not a IterDatapipe? I assume in prod then that the config is set to return the most recent data from database?

Reply:
well I could make it an `IterDatapipe`, was just trying to get the thing you wanted. You'll have to pass a configuration in, but I can help you make that

Reply:
Ill update the name